index.html

```diff
@@ -1,56 +1,220 @@
 </head>
 
 <body>
     <main>
-        <!-- The Header which includes the logo and hamburger menu -->
-        <header>
-            <a class="logo">My Logo</a>
-            <nav>
-                <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
-            </nav>
-        </header>
-        <!-- The Headline section that contains a description of the author -->
-        <section class="headline">
-                <h1>I'm Jerry </h1>
+
+        <div class="header-container">
+            <!-- The Header which includes the logo and hamburger menu -->
+            <header>
+                <a class="logo">My Logo</a>
+                <nav>
+                    <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+                </nav>
+            </header>
+
+            <!-- The Headline section that contains a description of the author -->
+
+            <section class="headline">
+                <h2>I'm Jerry </h2>
                 <h2>Glad to see <br> you!</h2>
-            <p class="about-my-self">I’m a software developer! I can help you build a product ,
-                 feature or website Look through some of my work and 
-                 experience! If you like what you see and have a project
-                  you need coded, don’t hestiate to contact me.</p>
+                <p class="my-profile">I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and
+                    experience! If you like what you see and have a project
+                    you need coded, don’t hestiate to contact me.</p>
                 <!-- Contacts -->
-             <a class="contact" href="#contacts">LET'S CONNECT</a>
+                <a class="contact" href="#contacts">LET'S CONNECT</a>
+                <!-- Social media links -->
+                <ul class="social-media">
+                    <li><a href="https://twitter.com/jerryowusu11">
+                            <i class="fa fa-twitter fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://linkedin.com/in/jeremiahowusu">
+                            <i class="fa fa-linkedin fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://medium.com/jerryowusu">
+                            <i class="fa fa-medium fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://github.com/jerryowusu">
+                            <i class="fa fa-github fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://angellist.com/jerryowusu">
+                            <i class="fa fa-angellist fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                </ul>
+            </section>
+        </div>
+        <!-- work section -->
+        <div class="works">
+            <section class="work-card"> 
```

Comment:
[OPTIONAL]
- It would be better to follow the next structure to improve our semantics:
```html
<section class="works">
   <div class="work-card"></div>
   <div class="work-card"></div>
   <div class="work-card"></div>
            ...
</section>
```